src/Microsoft.AspNetCore.SignalR.Protocols.MessagePack/Protocol/MessagePackHubProtocol.cs

```diff
@@ -29,13 +29,16 @@ public class MessagePackHubProtocol : IHubProtocol
 
         private static readonly string ProtocolName = "messagepack";
         private static readonly int ProtocolVersion = 1;
-
+        private static readonly int ProtocolMinorVersion = 0;
+        
         /// <inheritdoc />
         public string Name => ProtocolName;
 
         /// <inheritdoc />
         public int Version => ProtocolVersion;
 
+        public int MinorVersion => ProtocolMinorVersion;
```

Comment:
Add the `/// <inheritdoc />` line the other properties have. That tells our documentation generator to copy the docs from the `IHubProtocol2.MinorVersion` property here.